modules/tinymce/src/core/test/ts/browser/dom/SerializerTest.ts

```diff
@@ -163,8 +163,8 @@ describe('browser.tinymce.core.dom.SerializerTest', () => {
     DOM.setHTML('test', '<input type="text" value="text" length="128" maxlength="129" />');
     assert.equal(ser.serialize(DOM.get('test')), '<input type="text" value="text" length="128" maxlength="129" />');
 
-    DOM.setHTML('test', '<form method="post"><input type="hidden" name="method" value="get" /></form>');
-    assert.equal(ser.serialize(DOM.get('test')), '<form method="post"><input type="hidden" name="method" value="get" /></form>');
+    DOM.setHTML('test', '<form method="post"><input type="hidden" name="formmethod" value="get" /></form>');
```

Comment:
This was changed as it clobbers and as such was being removed. Also interesting to note the spec specifically mentions use "method" as a name should be avoided because it does clobber: https://html.spec.whatwg.org/#naming-form-controls:-the-name-attribute